saleor/plugins/webhook/tasks.py

```diff
@@ -66,7 +66,7 @@ class WebhookResponse:
 
 
 def create_deliveries_for_subscriptions(
-    event_type, subscribable_object, webhooks, requestor=None
+    event_type, subscribable_object, webhooks, requestor=None, sync_event=False
```

Comment:
Do you need this flag? 
Maybe you could just do something like this:
```
request=initialize_request(requestor, webhook.event in WebhookEventSyncType.ALL`),
```
Then you don't need to pass `sync_event` 🤗

Reply:
brilliant, thanks!